### PR TITLE
Personal Item: Attacus Fischer's Guitar

### PR DESCRIPTION
**Pull request by Coolsurf6**
## About the PR
Added a personal item for 'Attacus Fischer' for JustLeoHonestly

## Why / Balance
Personal Item.

## Technical details
Added new YML code based off the AcousticGuitarInstrument. I have also added a Jazz midi sounds using sound midi program 26.

## Media

Screenshot_20241117_171454

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added Attacus Fischer's Guitar. Red this time!

### CODE REVIEW

**Comment by Coolsurf6** [comment]
Baller

**Review by TsjipTsjip — APPROVED** [review]
GM approval. (Resprite rule)

File: Resources/Prototypes/_Umbra/Entities/Objects/Fun/Instruments/instruments_string.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+- type: entity
```
Comment (PursuitInAshes) [inline Resources/Prototypes/_Umbra/Entities/Objects/Fun/Instruments/instruments_string.yml:1]: Convention says you include the player name and character name at the start of this item. Refer to seren's guitar PR for an example.

File: Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "https://github.com/vgstation-coders/vgstation13 at 8d9c91e19cb52713c7f7f1804c2b6f7203f8d331, equipped backpack by ;3#1161",
```
Comment (PursuitInAshes) [inline Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json:8]: Is this copyright accurate?

**Review by PursuitInAshes — DISMISSED** [review]
Works in game but please address the other comments here.

File: Resources/Prototypes/_Umbra/Entities/Objects/Fun/Instruments/instruments_string.yml (hunk 1 above)
Comment (Coolsurf6) [reply Resources/Prototypes/_Umbra/Entities/Objects/Fun/Instruments/instruments_string.yml:1]: Will get this done ASAP. Not sure how I missed this.

Comment (Coolsurf6) [reply Resources/Prototypes/_Umbra/Entities/Objects/Fun/Instruments/instruments_string.yml:2]: Resolved with the same player comment as the Ace Bluewing's Guitar PR.

File: Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "wielded-inhand-right/left, inhand-right/left, icon, and equipped-SUITSTORAGE/BACKPACK modified by VividWisp from commit: https://github.com/space-wizards/space-station-14/commit/ae41d03ec6158e72450c9c045875a11e14fabd74.",
```
Comment (TsjipTsjip) [inline Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json:8]: See also Ace Bluewing's guitar.

I cannot find the original guitar sprite on this commit.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json (hunk 3 above)
Comment (Coolsurf6) [reply Resources/Textures/_Umbra/Objects/Fun/Instruments/attacusguitar.rsi/meta.json]: Check the Ace Bluewing's PR for the changes for this.

**Review by TsjipTsjip — APPROVED** [review]

**Pull request merged**